README.md

```diff
@@ -14,13 +14,25 @@ CMake version 3.12 is required to use these libraries.
 ### Development Environment
 Project contains files to work on Windows or Linux based OS.
 
+> Note: Project contains git submodules required to build. Before building run `git submodule update --init --recursive`

```

Comment:
ideally outline which submodules are required here, and how one may use their own version instead of the bundled one (important for package maintainers)